app/helpers/alchemy/admin/base_helper.rb

```diff
@@ -330,7 +330,7 @@ def new_asset_path_with_session_information(asset_type)
       # If you pass +'datetime'+ as +:type+ the datepicker will also have a Time select.
       # If you pass +'time'+ as +:type+ the datepicker will only have a Time select.
       #
-      # The date value gets localized via +I18n.l+. The format on Time and Date is +datepicker+, +timepicker+
+      # The format on Time and Date is +datepicker+, +timepicker+
```

Comment:
Is this comment still valid, now that we removed the `datepicker` and `timepicker` formats?

Reply:
Nope, removing.